GitUtils.ps1

```diff
@@ -282,15 +281,30 @@ function Find-Pageant() {
 
 # Attempt to guess $program's location. For ssh-agent/ssh-add.
 function Find-Ssh($program = 'ssh-agent') {
+    $sshLocation = 'C:\Program Files\Git\usr\bin\'

```

Comment:
This has to be given as preference from user perspective. not sure where to keep this and how to import that var here. temp hardcoded